plugins/inventory/terraform_state.py

```diff
@@ -26,11 +26,22 @@
     required: true
     type: str
     choices: [ cloud.terraform.terraform_state ]
-  backend_config:
+  backend_type:
     description:
-      - A Terraform backend configuration to an existing state file.
+      - The Terraform backend type from which the state file will be retrieved.
+      - Possible values include C(s3), C(remote), C(azurerm), C(local), C(consul), C(cos), C(gcs), C(http)
     type: str
     required: true
+  backend_config:
+    description:
+      - A group of key-values used to configure the backend.
+      - These values will be provided at init stage to the -backend-config parameter.
+    type: dict
+  backend_config_files:
+    description:
+      - The path to a configuration file to provide at init state to the -backend-config parameter.
+        This can accept a list of paths to multiple configuration files.
+    type: raw
```

Comment:
Could this be type `list` or better yet `pathlist`?

Reply:
updated to `list` with `elements: path`